docs/key_features.md

```diff
@@ -0,0 +1,26 @@
+# Key Features of `scores`
+
+## Data Handling
+
+- Works with labelled, n-dimensional data (e.g., geospatial, vertical and temporal dimensions) for both point-based and gridded data. `scores` can effectively handle the dimensionality, data size and data structures commonly used for:
+  - gridded Earth system data (e.g., numerical weather prediction models)
+  - tabular, point, latitude/longitude or site-based data (e.g., forecasts for specific locations).
+- Handles missing data, masking of data and weighting of results.
+- Supports [xarray](https://xarray.dev/) datatypes, and works with [NetCDF4](https://doi.org/10.5065/D6H70CW6), [HDF5](https://github.com/HDFGroup/hdf5), [Zarr](https://zarr.dev) and [GRIB](https://codes.wmo.int/grib2) data sources among others. 
+
+## Usability
+
+- A companion Jupyter Notebook [tutorial](project:./tutorials/Tutorial_Gallery.md) for each metric and statistical test that demonstrates its use in practice.
+- Novel scores not commonly found elsewhere (e.g., FIxed Risk Multicategorical (FIRM) score ([Taggart et al., 2022](https://doi.org/10.1002/qj.4266)), Flip-Flop Index ([Griffiths et al., 2019](https://doi.org/10.1002/met.1732), [2021](https://doi.org/10.1071/ES21010))). 
```

Comment:
Re. Flip-Flop Index link:
- Ideally I would link to the entry in the "included.md" table, but I don't know how to do that.
- Should it link to the paper reference (as is currently the case), or should it link to the API entry?

FIRM link:
- Ideally I would link to the entry in the "included.md" table, but I don't know how to do that.
- Should it link to the paper references (as is currently the case), or should it link to the API entry?

Reply:
@nicholasloveday can I confirm the correct term for FIRM is FIxed Risk Multicategorical (FIRM) Score?

I can't access the https://doi.org/10.1002/qj.4266 article, as it is paywalled.

But when I looked at the second version of the paper on arXiv (https://arxiv.org/pdf/2108.12814) - which I realise may not be identical to the final published version - there were two different descriptions of what FIRM stood for.

- On pages 3 and 7, it said "FIxed Risk Multicategory (FIRM) Forecast Framework"
- On page 15, it said "FIxed Risk Multicategorical (FIRM) framework"

So I just wanted to check that "FIxed Risk Multicategorical (FIRM) score" is correct (and that it should not be "FIxed Risk Multicategory (FIRM) score").

Reply:
@Steph-Chong I had to wait until today to chat to Rob. We can stick with "multicategorical".